app/utils.ts

```diff
@@ -37,3 +38,34 @@ export function getContract(
 
   return new Contract(address, ABI, getProviderOrSigner(library, account) as any);
 }
+
+// thanks @ajsantander -- (https://github.com/ajsantander/ovm-og-multisigwallet/blob/master/scripts/validate-owners.js)
+export const validateOwners = async ({ owners, required }) => {
+  owners = owners.split(',');
+  const encodedConstructorParameters = getEncodedConstructorParameters({ owners, required });
+  if (!checkBytesAreSafeForOvm(encodedConstructorParameters)) {
+    throw new Error('MultiSigWallet constructor parameters are not safe!');
```

Comment:
It'd be nice to give a more helpful error message here. Maybe something like

"One of the owner addresses contains a byte sequence that is unsafe as an OVM constructor parameter. Please remove the address here and add it to the multisig after it is deployed."

Reply:
way better!